### DISCUSSION TITLE
please help me.After installing the dietaryindex package, the library (dietaryindex) shows that the package does not exist, so nothing can be inputted

### DISCUSSION THREAD

**Discussion by wangdsds** [jamesjiadazhan/dietaryindex#248] [Q&A] [answered]
library(devtools)
install_github("jamesjiadazhan/dietaryindex")

library(dplyr)
library(haven)
library(readr)
library(dietaryindex)
setwd("C:/Users/CJL040/Desktop/new")
DEMO_PATH = read_xpt(file ='09-10human.XPT')
FPED_PATH_1 = read_sas('fped_dr1tot_0910.sas7bdat')
NUTRIENT_PATH_1 = read_xpt(file ='09-10nuXPT')

DII_NHANES_FPED(FPED_PATH=FPED_PATH_1, NUTRIENT_PATH=NUTRIENT_PATH_1, DEMO_PATH=DEMO_PATH)
HEI2015_NHANES_FPED(FPED_PATH=FPED_PATH_1, NUTRIENT_PATH=NUTRIENT_PATH_1, DEMO_PATH=DEMO_PATH)

Error in library(dietaryindex) : 不存在叫‘dietaryindex’这个名字的程辑包

**Comment by jamesjiadazhan** [comment, marked as answer]
Issue resolved in another post
